src/events.ts

```diff
@@ -24,8 +24,11 @@ export interface EventsApi {
 */
 export interface BatchEventsApi {
     batchCreate(
-        request?: CreateBatchEventsImportJobRequest,
-        jobOptions?: BatchJobOptions
+        request: {
+            body: CreateBatchEventsImportJobRequest,
+            includeSchema?: boolean,
+        },
+        jobOptions?: BatchJobOptions,
```

Comment:
I left `jobOptions` out of the `withOptions`, since it's lives on the job object and isn't really affect the "requests"